### ISSUE TITLE
Support for 1.15.2

### ISSUE THREAD

**Issue by jolibub** [McJty/RFToolsDimensions#285]
Issue description: Is this mod going to be updated to 1.15.2 ?

Steps to reproduce: 

Versions:
* Minecraft: 1.15.2
* Forge: 
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib:
* RFTools:
* RFTools Dimensions: 

Possibly incompatible mods: 

Relevant logs, if any:

**Comment by McJty** [comment]
Yes eventually. Could be relatively fast even

**Comment by Bass2style** [comment]
any news about this ? we have most content of rftools back now, but still no Dimensions, and it should be cool to see it back. ^^

**Comment by McJty** [comment]
Yeah sorry. Don't expect this soon. It's rather complicated and I'm preparing also how it should work in 1.16. So please don't wait for this

**Comment by Bass2style** [comment]
thanks for the reply and giving us news about it, more hyped to see your mods in 1.16 then. :D

**Comment by McJty** [comment]
I'm not saying RFToolsDimensions will not make it in 1.15.2. It is possible it will. I just haven't decided yet and it depends on how different 1.15.2 and 1.16.2 are in that aspect
